locksmith/src/utils/constants.ts

```diff
@@ -3,3 +3,4 @@ export const baseStripeFee = 30
 export const stripePercentage = 0.029
 export const GAS_COST = 200000 // hardcoded : TODO get better estimate, based on actual execution
 export const GAS_COST_TO_GRANT = 250000
+export const MIN_PAYMENT_STRIPE = 0.5 // 0.50 $
```

Comment:
@julien51 i see that you set this value as `100`, around the app we use `50 cents` as value? keep 1$ or 0.5$ here?

Reply:
<img width="795" alt="Screenshot 2023-05-12 at 21 55 22" src="https://github.com/unlock-protocol/unlock/assets/20865711/62e0f300-4af4-465d-8e97-02e117a1554f">

Reply:
I am confused? Where did I set 100?